.ci/jenkins_job_build.sh

```diff
@@ -10,6 +10,11 @@ set -o nounset
 set -o pipefail
 set -o errtrace
 
+# FIXME temp
+# export CI_JOB="CRI_CONTAINERD"
+export USE_DOCKER="true"
```

Comment:
we do not need this as this is configured at the jenkins job

Reply:
@GabyCT This PR is now WIP, I will ping you again when ready.